Automattic/jetpack
# Nav Unification: Force Simple sites to use Calypso Themes.

Regardless of Nav Unification, Simple sites cannot add their own themes, and should always be shown the WP.com Theme Showcase (from Calypso) – never the core `themes.php` page. This PR ensures the Appearance menu for WP.com Simple sites is always built with the `wp_admin_themes` value set to `false` (do not use the core theme page).

Fixes https://github.com/Automattic/wp-calypso/issues/51663

<img width="1585" alt="Screen Shot 2021-04-12 at 12 33 24 PM" src="https://user-images.githubusercontent.com/349751/114450913-7627a080-9b8b-11eb-889d-a157ce2cd4f2.png">

#### Jetpack product discussion
Nav Unification project thread: pbAPfg-Ou-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
This patch affects only Simple sites, so testing is best done with the diff and a WP.com sandbox.

* Apply D60049-code to your sandbox and sandbox the API.
* Load https://wordpress.com/me/account, and toggle on the "Show advanced dashboard pages" setting.
* Under My Sites, switch to a Simple site.
* Click on Appearance > Themes in the sidebar menu.
* Verify you are shown the Calypso theme showcase at the `wordpress.com/themes/*` URL.
